contracts/test/LON/Burn.t.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+
+import "contracts-test/LON/Setup.t.sol";
+
+contract TestLONBurn is TestLON {
+    using BalanceSnapshot for BalanceSnapshot.Snapshot;
+
+    function testCannotBurnMoreThanOwned() public {
+        lon.mint(user, 1e18);
+        uint256 excessAmount = lon.balanceOf(user) + 1;
+        vm.expectRevert("ERC20: burn amount exceeds balance");
+        vm.prank(user);
+        lon.burn(excessAmount);
+    }
+
+    function testBurn() public {
+        lon.mint(user, 1e18);
+        BalanceSnapshot.Snapshot memory userLon = BalanceSnapshot.take(user, address(lon));
+        vm.prank(user);
+        lon.burn(1e18);
```

Comment:
Minor suggestion : declare a `uint` to replace `1e18`.

Reply:
No problem! I think this's better! I have take the exact declared variable as the parameters of the functions.